pkg/shell/shell.go

```diff
@@ -39,6 +43,7 @@ func init() {
 	sshDev = os.Getenv("SSH_DEV")
 	sshDevHost = os.Getenv("SSH_DEV_HOST")
 	sshDevPort = os.Getenv("SSH_DEV_PORT")
+	SIGWINCH = regexp.MustCompile(`.*\[8;(.*);(.*)t`)
```

Comment:
Where did you source this regex from?

Reply:
I pieced it together from a few different webpages (e.g., lists of CSI codes and the xterm.js docs). The general idea is captured here: https://stackoverflow.com/questions/45775005/xterm-js-ws-ssh2-not-transmitting-terminal-resize-signals-sigwinch-to-ssh